blueocean-pipeline-api-impl/src/test/resources/io/jenkins/blueocean/rest/impl/pipeline/downstreamBuildLinksSeq.jenkinsfile

```diff
@@ -0,0 +1,47 @@
+pipeline {
```

Comment:
Mixes simple branch, single-stage named branch, and sequential stage branch for peace of mind.